fluentd/templates/service-extra.yaml

```diff
@@ -0,0 +1,29 @@
+{{- $root := . -}}
+{{- range $i, $service := .Values.extraService }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    {{- toYaml $service.annotations | nindent 4 }}
+  labels:
+    {{- include "fluentd.labels" $root | nindent 4 }}
+    {{- with $service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ include "fluentd.fullname" $root }}-{{ $service.name }}
+spec:
+  type: {{ $service.type }}
+  {{- if $root.Values.statefulset.enabled }}
+  clusterIP: ""
+  {{- end }}
```

Comment:
What does it mean for value to be empty ?

Reply:
Hmmm, I got an error if I didn't make ClusterIP an empty character when creating a service in StatefulSet.
However, this will not be reproduced.

Once, I will delete ClusterIP.